py3status/i3status.py

```diff
@@ -330,6 +330,10 @@ def write_tmp_i3status_config(self, tmpfile):
                         value = TZTIME_FORMAT
                     if key == "format_time":
                         continue
+                # Set output_format to i3bar for parsing regardless of what
+                # formatting we apply ourselves before printing
+                if key == "output_format":
+                    value = "i3bar"
```

Comment:
this code actually fixes a long standing bug in py3status, where setting output_format to anything other than i3bar causes the i3status integration to hang indefinitely or crash. 

py3status starts an i3status process internally, and parses its output to support i3status modules. This parsing requires the output to be the i3bar-compatible json format. if i3status produces any other output, which it will when output_format is set to anything other than i3bar, then the integration will break.

As a consequence, to make the output_format setting meaningful for py3status, it is necessary to overwrite the configured value in the generated config file before passing it to i3status.

Reply:
I get that. I was initially referring to modules having `output_format` config... which `khal_calendar` does have one, but then again, this doesn't write py3status modules in the config... so we're good here... then I wonder if it could be improved to only update the config inside `general {}` section instead. Either way, it's a good bug catch.